test/async.jl

```diff
@@ -0,0 +1,12 @@
+using DataDepsGenerators
+using Base.Test
+
+using ReferenceTests
+
+@testset "DataCite test" begin
```

Comment:
I think maybe this is not the right test set title.

Also the file name is wrong.

Async is not what you are testing here -- combining is.
It uses async, but not in a way you can test for.